mlflow/tracing/fluent.py

```diff
@@ -209,11 +219,12 @@ def get_traces(n: int = 1) -> List[Trace]:
 
 
 def search_traces(
-    experiment_ids: List[str],
+    experiment_ids: Optional[List[str]] = None,
     filter_string: Optional[str] = None,
     max_results: Optional[int] = None,
     order_by: Optional[List[str]] = None,
-):
+    extract_fields: Optional[List[str]] = None,
+) -> "pandas.DataFrame":
```

Comment:
Does `search_traces` have to work in mlflow-skinny (which doesn't include pandas as a dependency)?

Reply:
We don't any rule or policy for what must work in mlflow-skinny tho.

Reply:
Do you think you can make the call here? @harupy

Reply:
Let's add a try/catch around the import from within this API to make it clear that this API requires pandas to return a DF from the fluent API